### PR TITLE
Enhanced Navbar

### PR DESCRIPTION
**Pull request by Saatvik21**
# Related Issue
Fixes: #168 

# Prosposed Changes
- Added Transition on navbar for small screens.
- Fixed positioning of navbar for small screens
- Added transparency on hamburger menu

### CODE REVIEW

**Comment by garg3133** [comment]
Try this out:

```
@media screen and (max-width: 760px) {
	.navbar .container{
		flex-direction: column-reverse;
	    position: relative;
		overflow: visible;
	}

	.navbar .navbar-toggler {
		display: block;
	}

	.navbar .left-nav {
		align-self: flex-start;
		position: absolute;
		width: 125%;
		left: -12.5%;
		transform: translateY(100%);
		background: rgba(255, 140, 0, 0.9);
	}

	.navbar .left-nav .navbar-nav {
		flex-direction: column;
		overflow: hidden;
	}

	.navbar .left-nav .nav-item {
		width: 80%;
		margin: auto;
		transition: transform .5s ease;
	}

	.navbar .nav-collapsed {
		height: 0;
	}

	.navbar .nav-collapsed .nav-item {
		transform: translateY(200%);
	}

	.navbar .right-nav {
		align-self: flex-end;
	}
}
```

**Comment by garg3133** [comment]
Instead of making the whole container absolute, I've just made the `left-nav` absolute, which allows me to make the background of just `left-nav` transparent, along with no need to insert padding on right and left of different components.

**Comment by Saatvik21** [comment]
> Instead of making the whole container absolute, I've just made the `left-nav` absolute, which allows me to make the background of just `left-nav` transparent, along with no need to insert padding on right and left of different components.

Yes, this looks better! 

And sir what about the transition of the dropdown menu? Is that fine? 

And should I update my PR with the code you just provided.

**Comment by garg3133** [comment]
> And sir what about the transition of the dropdown menu? Is that fine?

It looks fine, but a transition like before would have been better. See if you can implement that.

> And should I update my PR with the code you just provided.

Yeah! And also remove that `open` class you added from JS,.

**Comment by Saatvik21** [comment]
> It looks fine, but a transition like before would have been better. See if you can implement that.

Sir I tried to implement that before as well but couldn't do it.

File: static/home/css/style.css [hunk 1]
```diff
@@ -142,8 +142,10 @@ a:hover{
 }
 
 @media screen and (max-width: 760px) {
-	.navbar .container {
+	.navbar .container{
 		flex-direction: column-reverse;
+	    position: relative;
```
Comment (garg3133) [inline static/home/css/style.css:147]: Fix the indentation here (use `tab` for indentation instead of `space`.

File: static/home/css/style.css [hunk 2]
```diff
@@ -152,23 +154,40 @@ a:hover{
...
+	}
+
+	.navbar .right-nav {
+		align-self: flex-end;
+	}
 }
 
 
+
+
```
Comment (garg3133) [inline static/home/css/style.css:190]: Remove these two extra lines

**Review by garg3133 — COMMENTED** [review]
Can you squash your commits after making the below corrections? Rest all looks good to me.

**Comment by garg3133** [comment]
Merged! 🎉 

Thanks, @Saatvik21 for your contribution! 🚀

**Pull request merged**